src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/page/CommentTabPanel.kt

```diff
@@ -38,7 +38,7 @@ class CommentTabPanel(
         val comments = driver.findElements(By.cssSelector("#issue_actions_container > div.activity-comment"))
         return comments.map {
             val id = it.getAttribute("id").split("-")[1]
-            val url = it.findElement(By.cssSelector(".comment-created-date-link")).getAttribute("href")
+            val url = it.findElement(By.cssSelector(""".action-head a[href^="/browse/"]""")).getAttribute("href")
```

Comment:
We can add Jira version support as a behavioral contract (of `getComments` or `CommentTabPanel` or the whole lib).

Reply:
What do you mean?

Reply:
```java
/**
 * Represents an issue comment tab panel from Jira 8.5.0-8.17.0.
 */
class CommentTabPanel(
```

Reply:
Fair enough